app/assets/stylesheets/ama_layout/layout_components/callouts.scss

```diff
@@ -56,13 +56,11 @@
 }
 
 .promo-block{
-  @include callout($ash);
```

Comment:
@kaytcampbell The plan is to remove the grey bg on both since now that they are in their own sections they dont really need the grey bg to break them out. 

Registries will look like this now:
![screen shot 2016-08-17 at 11 53 26 am](https://cloud.githubusercontent.com/assets/9700058/17747275/493c29f2-6471-11e6-9608-b502d3178424.png)